config/initializers/menus.rb

```diff
@@ -36,10 +36,17 @@
             if: Proc.new { User.current.logged? }
   # projects menu will be added by
   # Redmine::MenuManager::TopMenuHelper#render_projects_top_menu_node
-  menu.push :administration,
-            { controller: '/admin', action: 'projects' },
-            if: Proc.new { User.current.admin? },
-            last: true
+  menu.push :work_packages,
+            { controller: '/work_packages' },
+            caption: I18n.t('label_work_package_plural'),
+            if: Proc.new { User.current.allowed_to?(:view_work_packages, nil, global: true) }
+  menu.push :news,
+            { controller: '/news' },
+            if: Proc.new { User.current.allowed_to?(:view_news, nil, global: true) }
+  menu.push :time_sheet,
+            { controller: '/time_entries' },
+            caption: I18n.t('label_time_sheet_menu'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.